op-node/rollup/superchain.go

```diff
@@ -86,6 +86,7 @@ func LoadOPStackRollupConfig(chainID uint64) (*Config, error) {
 		DeltaTime:              chConfig.DeltaTime,
 		EcotoneTime:            chConfig.EcotoneTime,
 		FjordTime:              chConfig.FjordTime,
+		GraniteTime:            chConfig.GraniteTime,
```

Comment:
No holocene here?

Reply:
It's still missing in the SCR. Will be added later. Right now, we just need to unblock granite.